docs/development/visualize/development-create-visualization.asciidoc

```diff
@@ -335,7 +336,36 @@ This response matches the visData property of the <visualization> directive.
 
 [[development-default-response-handler]]
 ==== default response handler
-Default response handler will convert pure elasticsearch response to tabular format.
+Default response handler will convert pure elasticsearch response to tabular format. Its the recommended response
```

Comment:
`The default response handler converts pure elasticsearch responses into a tabular format. It is the recommended responseHandler.`